lib/ActivityStreams.js

```diff
@@ -5,15 +5,18 @@
 const {Cu} = require("chrome");
 const {data} = require("sdk/self");
 const {PageMod} = require("sdk/page-mod");
+const {setTimeout, clearTimeout} = require("sdk/timers");
 const {ActionButton} = require("sdk/ui/button/action");
 const tabs = require("sdk/tabs");
 const {PlacesProvider} = require("lib/PlacesProvider");
 const {SearchProvider} = require("lib/SearchProvider");
 const {TabTracker} = require("lib/TabTracker");
+const {PreviewProvider} = require("lib/PreviewProvider");
 const {TelemetrySender} = require("lib/TelemetrySender");
 const {PerfMeter} = require("lib/PerfMeter");
 const {AppURLHider} = require("lib/AppURLHider");
 const am = require("content-src/actions/action-manager");
+const PRIME_CACHE_TIMEOUT = 216000; // every 6 hours, prime the cache
```

Comment:
Is this value in milliseconds?

According to [MDN `setTimeout()` docs](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setTimeout), the `delay` should be specified in milliseconds.
According to Google's conversion, 216000ms is 0.06h (or 3.6 minutes).
According to https://www.unitjuggler.com/convert-time-from-h-to-ms.html?val=6, 6 hours is 21600000 ms.


Not sure if the comment is wrong, or the value, or maybe I'm missing something else.